### PR TITLE
feat/docsite publishing

### PR DESCRIPTION
**Pull request by scrthq**
- **feat: #comment updated mkdocs related configs, no code changes**
- **feat: #comment forcing LF line ending swap on requirements.txt file**
- **chore: #comment requirements.txt cleanup after line-ending fix**
- **feat: #comment adding additional docs**

### CODE REVIEW

**Review by climbertjh2 — CHANGES_REQUESTED** [review]
Minor fixes required.

File: docs/content/index.md [hunk 1]
```diff
@@ -106,36 +214,60 @@ SYNOPSIS:
 OPTIONS:
         -v | --version           Prints version number.
 
-        -p | --preserve-report   Add timestamp to the final report file to avoid overwriting it after multiple executions.
+        -p | --preserve-report   Add timestamp to the final report file to avoid overriding it after multiple executions.
```
[the remark this answers is not in the document]
Comment (scrthq) [reply docs/content/index.md:217]: thanks for catching these index.md changes! not sure how these were reverted in this change, but very valid callout.

File: docs/content/index.md [hunk 2]
```diff
@@ -106,36 +214,60 @@ SYNOPSIS:
...
         --source-dir             Path to the directory containing the code/files you wish to scan. Defaults to $(pwd)
         --output-dir             Path to the directory that will contain the report of the scans. Defaults to $(pwd)
         --ext | -extension       Force a file extension to scan. Defaults to identify files automatically.
         --force                  Rebuild the Docker images of the scanning tools, to make sure software is up-to-date.
-        --no-cleanup             Don't cleanup the work directory where temp reports are stored during scans.
-        --debug                  Print ASH debug log information where applicable.
```
[the remark this answers is not in the document]
Comment (scrthq) [reply docs/content/index.md:115]: This should not have been removed! Good catch, fixing.

File: docs/content/tutorials/CI/Jenkins/Jenkinsfile [hunk 3]
```diff
@@ -1,36 +1,29 @@
...
 
       # Run the ASH container image
-      docker run \
-        -v="$(pwd):/src" \
-        -v="$(pwd)/${ASH_OUTPUT_PATH}:/out" \
-        ${ASH_IMAGE_REPO}:${ASH_IMAGE_TAG} \
-        ash \
-        --source-dir /src \
-        --output-dir /out
+      export PATH="$(pwd)/../automated-security-helper:$PATH"
```
[the remark this answers is not in the document]
Comment (scrthq) [reply docs/content/tutorials/CI/Jenkins/Jenkinsfile:26]: yes indeed, thank you!

**Comment by climbertjh2** [comment]
This looks good - approving.

**Review by climbertjh2 — APPROVED** [review]

**Pull request merged**